docs/how-to/rocks/index.rst

```diff
@@ -5,6 +5,7 @@
 
    Use Rockcraft's GitHub Action <rockcraft-pack-action>
    Convert an entrypoint to a Pebble layer <convert-to-pebble-layer>
+   Add a new internal user in a rock <add-internal-user-in-rocks>
```

Comment:
```suggestion
   Add a new internal user to a rock <add-internal-user-in-rocks>
```

Missed this on my first review. Any chance we can match the file name to the title, too?